content/guides/integration-guides/cinder-compatible-SODA-API/_index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Cinder Compatible 
+menuTitle: "SODA API"
+description: "A guide for enable Cinder Compatible SODA API."
+weight: 1
+disableToc: false
+tags: ["Cinder Compatible SODA API"] 
```

Comment:
The tags are usually common for the section. 
Please use the following:
`tags: [integration guide, cinder]`